src/cache/cache_operations.rs

```diff
@@ -179,6 +182,7 @@ impl RequestCached {
             cache_duration: request_cache_duration(),
             error_cache_duration: request_error_cache_duration(),
             cache_all_errors: false,
+            headers: HashMap::new(),
```

Comment:
nitpick: Not sure if `Default::default()` is more idiomatic in this scenario.

Reply:
Scratch that 😅 it's totally possible, I agree I would prefer that too 👍